src/Files.App/ViewModels/UserControls/AddressToolbarViewModel.cs

```diff
@@ -339,7 +339,12 @@ public async Task PathBoxItem_Drop(object sender, DragEventArgs e)
 
 			await signal.WaitAsync();
 
-			deferral.Complete();
+			try
+			{
+				deferral.Complete();
+			}
+			catch { }
```

Comment:
Most of the exceptions that occur in deferral are caused by Complete() being called even though there is an operation in progress. So we should make sure that all async operations are awaited correctly.
Even if an exception is caught here and a crash is avoided, the drop operation will not complete successfully and may behave unexpectedly.

Reply:
I think the crashes are occurring because the PathBoxItemDropped event invoked at L332 can't be awaited.

Reply:
I pushed some changes in [dcd1724](https://github.com/files-community/Files/pull/16048/commits/dcd17246901064a03f0c60cf146fa15d731ead4d)

Reply:
The similarity between this and #16026 is that both call FilesystemHelpers.PerformOperationTypeAsync() internally. So I suspect the problem is in this method.

Reply:
This may be the cause of the crash.
https://github.com/files-community/Files/blob/660a51e258ed4956680fa0aa6419da2b32144019/src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs#L478